client-js/app/elements/labrad-registry.html

```diff
@@ -6,40 +6,188 @@
+      <menu-button data-dialog="doDelete" name="delete" icon-name="delete" class = "done"></menu-button>
+      <menu-button data-dialog="newFolderDialog" name="new folder" icon-name="folder-open" class = "done"></menu-button>
+      <menu-button data-dialog="copyDialog" name="copy" icon-name="content-copy" class = "done"></menu-button>
+      <menu-button data-dialog="renameDialog" name="rename" icon-name="code" class= "done"></menu-button>
+      <menu-button data-dialog="newKeyDialog" name="new key" icon-name="create"  class = "done"></menu-button>
+     </section>
     </div>
+    <div id="reg-area"> 
+      <table>
+          <tbody class="table-header" is="selectable-table"> 
+            <th></th>
+            <th>Key/Dir Name</th>
+            <th>Key Value</th>
+          </tbody>
+          <tbody is="selectable-table"  id="dirList" attr-for-selected="name" selected="{{selDir}}" on-tap="selectDir">       
+            <template is="dom-repeat" items="{{dirs}}">
+                <tr name="{{item.name}}">
+                  <td>
+                    <iron-icon icon = "more-vert"></iron-icon>
+                  </td>
+                  <td>
+                   <a is="app-link" path="{{item.url}}" href="{{item.url}}">
+                     <dir-item kind="folder">{{item.name}}</dir-item>
+                    </a>
+                  </td>
+                  <td></td>
+                </tr>
+            </template>
+          </tbody>
+          <tbody is="selectable-table" is="keyList" attr-for-selected="name"  selected="{{selKey}}" on-tap="selectKey"> 
+            <template is="dom-repeat" items="{{keys}}">
+              <tr name="{{item.name}}">
+                <td>
+                  <iron-icon icon = "more-vert"></iron-icon>
+                </td>
+                <td>
+                  <dir-item kind="key">{{item.name}}</dir-item>
+                </td>
+                <td>
+                  <form is="iron-form" id="{{item.name}}">
+                      <iron-a11y-keys keys="tab" on-keys-pressed="incrementSelector">
+                       <paper-input-container>
+                        <label>{{item.value}}</label>
+                        <input is="iron-input" name="{{item.name}}">
+                       </paper-input-container>
+                      </iron-a11y-keys>
+                  </form>
+                </td>
+              </tr>
+            </template>
+          </tbody>
+      </div>
+    </table>
+    <!-- These are the dialogs -->
+    <paper-dialog id="doRename">
+      <p>Rename "<span>{{selItem}}</span>" to something else</p>
+      <form>
+        <input></input>
+      </form>
+      <div class="buttons">
+        <paper-button dialog-confirm autofocus>Submit</paper-button>
+      </div>
+    </paper-dialog>
+
+    <paper-dialog id="doDelete">
+      <p>Delete "<span class="nothing">{{selDir}}</span><span class="nothing">{{selKey}}</span>" of type <span>{{selectType}}</span>?</p>
```

Comment:
Vestige from debugging. I was having trouble formatting this, but fixed it and forgot to take the class away.